nodejs/week3/api/contacts.js

```diff
@@ -0,0 +1,42 @@
+import express from 'express';
+import database from "../database.js";
+
+const router = express.Router();
+const apiRouter = express.Router();
+const contactsAPIRouter = express.Router();
+
+// http://localhost:3000/api/contacts/
+// http://localhost:3000/api/contacts/?sort=first_name%20ASC
+// http://localhost:3000/api/contacts/?sort=last_name%20DESC
+
+contactsAPIRouter.get("/", async (req, res) => {
+    let query = database.select("*").from("contacts");
+   
+    if ("sort" in req.query) {
+        const validTags = ["first_name ASC", "first_name DESC", "last_name ASC", "last_name DESC"]
```

Comment:
This should be case insenstive - means small case asc and desc should also work.

Reply:
Made sort case sensitive.